### DISCUSSION TITLE
Download Setup for WIN-Companion App - Defender Alert

### DISCUSSION THREAD

**Discussion by expressaccount** [akopachov/flipper-zero_authenticator#247] [Q&A] [answered]
When I download the app for Windows, Windows Defender tells me it has found potentially unwanted software. It says the app contains PUABundler:Win32/CandyOpen which is considered potentially dangerous. Have the downloads been compromised or is there an explanation for this?

**Comment by akopachov** [comment]
You have to use official repo and [releases](https://github.com/akopachov/flipper-zero_authenticator-companion/releases). There, each release is getting automatically scanned by Virus Total and appropriate reports are publicly available. Also, since this is open source software, you can always build it yourself, to make sure nothing is modified anywhere.

Other than that, from time to time anti-virus software can report false-positives. Usually it worth to double check such a cases by other anti-virus software.

**Reply by expressaccount** [reply]
I downloaded it from the official site. I have now downloaded it again from your link, but Windows Defender tells me again that it has found the threat PUABundler:Win32/CandyOpen. Unfortunately I am too stupid to compile it myself. The Defender speaks of a low risk, it may well be that there are reasons for this library or what ever this bundler is, because certain functions of the app require it.

**Reply by akopachov** [reply]
Well, than you may do one of two things:

1. Rely on Windows Defender, ignoring the fact that every anti-virus software may report false-positives, and just avoid using the software.
2. Do some research, by testing given software using other anti-virus software (using Virus Total for example), accept the fact that each anti-virus software may report false-positives.

Make a choice and do whatever is more preferable for you.

**Reply by expressaccount** [reply]
I did the second one. But when I clicked on the Virus Total link on the [release page](https://github.com/akopachov/flipper-zero_authenticator-companion/releases) and even there I get a message that it is potentially dangerous. That's exactly why I got suspicious. [VirusTotalResult](https://www.virustotal.com/gui/file/3dbbcd84f72fae794cd1af721634346e17ae6b1a26525302a04ec81cff2e2166)

So far, i think i let it be...

**Reply by akopachov** [reply, marked as answer]
It was reported by only one out of 69 antiviruses available in Virus Total. And that one is Bkav Pro - antivirus from Vietnam company that I never heard about before. Also, quick googling shown that it reports a lot of similar false-positives.

Tbh, I don't feel there is something to worry about. But, as I said before - make your own choice and do whatever is more preferable for you.
